modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -983,6 +986,19 @@ private void _deleteObjectFields(
 		}
 	}
 
+	private void _deleteObjectRelationshipsByParameterFieldId(
```

Comment:
I think the method name should be: `_deleteObjectRelationshipByParameterFieldId()`
I found 2 examples in Objects: `deleteObjectFieldByObjectDefinitionId()` / `deleteObjectFolderItemByObjectDefinitionId()`. The name of the entity is in the singular :+1: